gridcomps/ExtData/ExtDataGridCompMod.F90

```diff
@@ -3254,15 +3254,15 @@ subroutine MAPL_ExtDataInterpField(item,time,field,vector_comp,rc)
            nymd2=0
         end if
         
-        if (lgr%isEnabledFor(DEBUG) .and. .not. item%doInterpolate) then
-           call lgr%debug('   MAPL_ExtDataInterpField: Uninterpolated field %a set to sample L %i0.8 %i0.6', trim(item%name),  nymd1, nhms1)
-        else if (time == item%interp_time1) then
-           call lgr%debug('   MAPL_ExtDataInterpField: Interpolated field %a set to sample L %i0.8 %i0.6', trim(item%name),  nymd1, nhms1)
-        else if (time == item%interp_time2) then
-           call lgr%debug('   MAPL_ExtDataInterpField: Interpolated field %a set to sample R %i0.8 %i0.6', trim(item%name),  nymd2, nhms2)
-        else
-           call lgr%debug('   MAPL_ExtDataInterpField: Interpolated field %a between %i0.8 %i0.6 and %i0.8 %i0.6 (%f10.6 fraction)', trim(item%name), nymd1, nhms1, nymd2, nhms2, alpha)
-        end if
+        !if (lgr%isEnabledFor(DEBUG) .and. .not. item%doInterpolate) then
+           !call lgr%debug('   MAPL_ExtDataInterpField: Uninterpolated field %a set to sample L %i0.8 %i0.6', trim(item%name),  nymd1, nhms1)
+        !else if (time == item%interp_time1) then
+           !call lgr%debug('   MAPL_ExtDataInterpField: Interpolated field %a set to sample L %i0.8 %i0.6', trim(item%name),  nymd1, nhms1)
+        !else if (time == item%interp_time2) then
+           !call lgr%debug('   MAPL_ExtDataInterpField: Interpolated field %a set to sample R %i0.8 %i0.6', trim(item%name),  nymd2, nhms2)
+        !else
+           !call lgr%debug('   MAPL_ExtDataInterpField: Interpolated field %a between %i0.8 %i0.6 and %i0.8 %i0.6 (%f10.6 fraction)', trim(item%name), nymd1, nhms1, nymd2, nhms2, alpha)
+        !end if
```

Comment:
Can this commented out code be deleted?   Or why is ic commented out?

Reply:
I was debugging and I remember comment that out as the output was overwhelming, I'll undelete it.